pkg/watcher/watcher.go

```diff
@@ -33,26 +33,31 @@ import (
 
 	"github.com/francoispqt/gojay"
 	log "github.com/sirupsen/logrus"
+
+	statistics "github.com/grd/stat"
 )
 
 const (
-	baseUrl        = "/watcher"
-	FifteenMinutes = "15m"
-	TenMinutes     = "10m"
-	FiveMinutes    = "5m"
-	CPU            = "CPU"
-	Memory         = "Memory"
+	targetLoad             = "/watcher"
+	variationRiskBalancing = "/variation"
+	FifteenMinutes         = "15m"
+	TenMinutes             = "10m"
+	FiveMinutes            = "5m"
+	CPU                    = "CPU"
+	Memory                 = "Memory"
+	AVG                    = "AVG"
+	STD                    = "STD"
+	minute                 = 15
+	cacheSize              = 15
 )
 
 type Watcher struct {
-	mutex         sync.RWMutex // For thread safe access to cache
-	fifteenMinute []WatcherMetrics
-	tenMinute     []WatcherMetrics
-	fiveMinute    []WatcherMetrics
-	cacheSize     int
-	client        FetcherClient
-	isStarted     bool // Indicates if the Watcher is started by calling StartWatching()
-	shutdown      chan os.Signal
+	mutex       sync.RWMutex // For thread safe access to cache
+	minuteQueue []WatcherMetrics
```

Comment:
I believe the rolling window should be configurable, at least giving several options.

Reply:
ok, it should be configurable, let's add some envs to loadwatcher?